.ocamlformat

```diff
@@ -0,0 +1,70 @@
+profile=ocamlformat
```

Comment:
I think you need to add a version here too. `version=0.14.1`.
Can you also add the latest ocamlformat to xs-opam in the same place merlin is?

Reply:
sure thing